plugins/inputs/kube_inventory/kube_inventory.go

```diff
@@ -67,6 +70,14 @@ func (ki *KubernetesInventory) Init() error {
 	if err != nil {
 		return err
 	}
+	if ki.ResponseTimeout < config.Duration(time.Second) {
+		ki.ResponseTimeout = config.Duration(time.Second * 5)
+	}
+	ki.httpClient, err = newHTTPClient(ki.ClientConfig, ki.BearerToken, ki.ResponseTimeout)
```

Comment:
Should we only create this if KubeletURL != ""? To avoid any extra resources, even if they are minimal, and avoid causing any new errors with existing configs that do not use this?

Reply:
good idea - done